tyche-update-helper.php

```diff
@@ -34,3 +35,24 @@ function ts_modify_curl_request( &$handle, $args, $url ) {
 }
 
 add_action( 'http_api_curl', 'ts_modify_curl_request', 10, 3 );
+
+/**
+ * Function orddd_remove_transient_from_rest_api.
+ *
+ * @version 2.0.0
+ */
+function orddd_remove_transient_from_rest_api() {
+	$orddd_is_transient_removed = get_option('orddd_is_transient_removed');
+	if ( ! $orddd_is_transient_removed ) {
+		$fname  = ABSPATH . '/wp-content/plugins/order-delivery-date/plugin-updates/EDD_SL_Plugin_Updater.php';
+		$fhandle = fopen( $fname, "r" );
+		$content = fread( $fhandle, filesize( $fname ) );
+		$content = str_replace( '&& ! $plugin_transient', '', $content );
+		$fhandle = fopen( $fname, "w" );
+		fwrite( $fhandle, $content );
+		fclose( $fhandle );
+		update_option('orddd_is_transient_removed');
```

Comment:
update_option needs two parameters. Please add the value that needs to be updated